templates/intro-tune/filesystem_utils.py

```diff
@@ -0,0 +1,22 @@
+from s3fs import S3FileSystem
+import pyarrow.fs
+from typing import Optional, Tuple
+
+
+def get_path_and_fs(storage_path: str) -> Tuple[str, Optional[pyarrow.fs.FileSystem]]:
+    """Get the final storage path and corresponding filesystem, if applicable.
+
+    For S3, this will use s3fs.
+    For others, this will use the default filesystem(None).
+    """
+    s3_prefix = "s3://"
+
+    if storage_path.startswith(s3_prefix):
```

Comment:
does this work for GCP?

Reply:
Yep this is customization specific to S3 + Anyscale Hosted.